### PR TITLE
NF: improve the readability of some strings

### PR DESCRIPTION
**Pull request by Arthur-Milchior**
Specifically, the ones that represents json, sql or js. I found they used to be hard to read, and I expect to lower this issue.

### CODE REVIEW

File: AnkiDroid/src/test/java/com/ichi2/anki/dialogs/CustomStudyDialogTest.kt [hunk 1]
```diff
@@ -85,23 +85,24 @@ class CustomStudyDialogTest : RobolectricTest() {
             customStudy.remove("mod")
             customStudy.remove("name")
             val expected =
```
Comment (david-allison) [inline AnkiDroid/src/test/java/com/ichi2/anki/dialogs/CustomStudyDialogTest.kt:87]: `@Language("json")`

File: AnkiDroid/src/main/java/com/ichi2/libanki/Notetypes.kt [hunk 2]
```diff
@@ -709,11 +709,7 @@ class Notetypes(
             }
 
         private const val DEFAULT_TEMPLATE =
```
Comment (david-allison) [inline AnkiDroid/src/main/java/com/ichi2/libanki/Notetypes.kt:711]: `@Language("json")`

File: AnkiDroid/src/main/java/com/ichi2/anki/Info.kt [hunk 3]
```diff
@@ -138,12 +138,17 @@ class Info :
                          */
                             @Suppress("ktlint:standard:max-line-length")
                             webView.loadUrl(
-                                "javascript:document.body.style.setProperty(\"color\", \"" + textColor + "\");" +
-                                    "x=document.getElementsByTagName(\"a\"); for(i=0;i<x.length;i++){x[i].style.color=\"" + anchorTextColor +
-                                    "\";}" +
-                                    "document.getElementsByTagName(\"h1\")[0].style.color=\"" + textColor + "\";" +
-                                    "x=document.getElementsByTagName(\"h2\"); for(i=0;i<x.length;i++){x[i].style.color=\"#E37068\";}" +
-                                    "document.body.style.setProperty(\"background\", \"" + background + "\");",
+                                """javascript:document.body.style.setProperty("color", "$textColor");
```
Comment (david-allison) [inline AnkiDroid/src/main/java/com/ichi2/anki/Info.kt:141]: Has this been tested?

File: AnkiDroid/src/main/java/com/ichi2/anki/MetaDB.kt [hunk 4]
```diff
@@ -71,16 +71,27 @@ object MetaDB {
 
         // Create tables if not exist
         metaDb.execSQL(
-            "CREATE TABLE IF NOT EXISTS languages (" + " _id INTEGER PRIMARY KEY AUTOINCREMENT, " +
-                "did INTEGER NOT NULL, ord INTEGER, " + "qa INTEGER, " + "language TEXT)",
+            """CREATE TABLE IF NOT EXISTS languages (
```
Comment (david-allison) [inline AnkiDroid/src/main/java/com/ichi2/anki/MetaDB.kt:74]: Have all the changes to this file been tested (or are they under coverage)?

**Review by david-allison — APPROVED** [review]
I feel this introduces unnecessary risk: we're changing stable, untested code for no functionality improvements and minor readability improvements with very little testing

I'd be much happier if integration tests were added in future which impact the changed lines, as those both give evidence that the changes are correct, and improve the safety of the codebase

But... it's written, it's probably right, and we're at the start of an alpha cycle, let's get it in

**Review by lukstbit — APPROVED** [review]
Looks better, thanks!
I would have gone even further to improve the formatting for some of the sql strings(upper casing keywords, extra indentation).

**Pull request merged**